Sources/Client/Utils/RepeatingTimer.swift

```diff
@@ -16,19 +16,19 @@ final class RepeatingTimer {
     
     typealias EventHandler = () -> Void
     
-    private let queue: DispatchQueue?
+    private let queue: DispatchQueue
     private var state: State = .suspended
     let timeInterval: DispatchTimeInterval
-    var eventHandler: EventHandler?
+    var eventHandler: EventHandler
```

Comment:
Why don't we keep eventHandler as optional if we're going to set it to `{}`?

Reply:
I updated. Please refresh.